study-builder/studies/snippets/picklist-question-multi-checklist.conf

```diff
@@ -0,0 +1,14 @@
+{
+  "questionType": "PICKLIST",
+  "selectMode": "MULTIPLE",
+  "renderMode": "CHECKBOX_LIST",
```

Comment:
There is already the `picklist-question-multi-list.conf` snippet, does that not meet your needs? Note that multiple-choice picklist should already be rendered as checkboxes when specified with `LIST` render mode. `CHECKBOX_LIST` is a way to force rendering checkboxes, say, for a single-choice picklist.

Reply:
That makes sense!  I was thinking it would be bullets or something.